src/harness/util.py

```diff
@@ -285,3 +288,61 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
   request['cpiSignatureData']['protectedHeader'] = jwt_message[0]
   request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
   request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
+     
+def computePropagationAntennaModel(request):
+    reliability_level = request['reliabilityLevel']
+    if reliability_level not in [-1, 0.05, 0.95]:
+        response = 400
+        return response
+    tx = request['cbsd']
+    if ('fss' in request) and ('ppa' in request):
+        response = 400
+        return response
+    elif 'ppa' in request:
+        rx ={}
+        rx['height'] = 1.5
+        isfss = False
+        coordinates = [];
+        ppa = request['ppa']
+
+        ARCSEC = 2
+        ppa_points = ComputePPAProtectionPoints(ppa['geometry']['coordinates'], ARCSEC)
```

Comment:
To use the new function in geo/utils.py,  simply do:
```
  ppa_points = utils.GridPolygon(ppa['geometry'], ARCSEC)
```
The routine will take care of all possible geometry (simple polygon, polygon with holes, multi-polygon with holes).